k8s/test-validation.py

```
@@ -0,0 +1,318 @@
+        }
+
+    def validate_yaml_syntax(self, file_path: Path) -> bool:
+        """Validate YAML syntax"""
+        try:
+            with open(file_path, 'r', encoding='utf-8') as f:
+                yaml.safe_load_all(f)
+            return True
+        except yaml.YAMLError as e:
+            self.errors.append(f"YAML syntax error in {file_path}: {e}")
+            return False
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)